app/assets/stylesheets/base/flexbox.scss

```diff
@@ -0,0 +1,119 @@
+@import '../vars/variables';
+
+// Flex flow
+@include generate-classes(
+  'flex',
+  'flex-flow',
+  (
+    'row': row,
+    'col': column,
+    'row-reverse': row-reverse,
+    'col-reverse': column-reverse,
+  )
+);
+
+// Flex wrap
+@include generate-classes(
+  'flex',
+  'flex-wrap',
+  (
+    'wrap': wrap,
+    'nowrap': nowrap,
+    'wrap-reverse': wrap-reverse,
+  )
+);
+
+// Align items
+@include generate-classes(
+  'items',
+  'align-items',
+  (
+    'stretch': stretch,
+    'start': flex-start,
+    'center': center,
+    'end': flex-end,
+    'baseline': baseline,
+  )
+);
+
+// Align content
+@include generate-classes(
+  'content',
+  'align-content',
+  (
+    'start': flex-start,
+    'center': center,
+    'end': flex-end,
+    'between': space-between,
+    'around': space-around,
+  )
+);
+
+// Align self
+@include generate-classes(
+  'self',
+  'align-self',
+  (
+    'auto': auto,
+    'start': flex-start,
+    'center': center,
+    'end': flex-end,
+    'stretch': stretch,
+  )
+);
+
+// Justify content
+@include generate-classes(
+  'justify',
+  'justify-content',
+  (
+    'start': flex-start,
+    'center': center,
+    'end': flex-end,
+    'between': space-between,
+    'around': space-around,
+  )
+);
+
+// Flex
+@include generate-classes(
+  'flex',
+  'flex',
+  (
+    'initial': 0 1 auto,
```

Comment:
Isn't initial by default `0 auto`? See https://css-tricks.com/almanac/properties/f/flex/#article-header-id-1. Or is this what we consider our initial value?

Reply:
`flex: 0 auto` is shorthand for `flex: 0 1 auto`  - i got used to writing longer version because shorter one was causing errors in IE10  :D i guess it's finally a time to change my habits :D